pyproject.toml

```diff
@@ -34,8 +34,17 @@ dependencies = [
     "hvac",
     "packaging",
     "typing-extensions>=4.5.0",
-    "backports.strenum>=1.3.1",
+    'backports.strenum>=1.3.1; python_version < "3.11"',
 ]
+[project.optional-dependencies]
+dev = [
+        "typing-inspect",
+        "pytest",
+        "pytest-asyncio",
+        "Twine",
+        "setuptools",  # ?
+        "pylxd",
+    ]
```

Comment:
Needs dedenting. I'll have to review the actual changes on Monday. Good luck, have fun!

Reply:
done thx